src/MsSqlMicrosoftDataClientSqlTransportAcceptanceTests/MsSqlMicrosoftDataClientSqlTransportAcceptanceTests.csproj

```diff
@@ -13,17 +13,17 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.4" />
-    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="8.0.0-alpha.842" GeneratePathProperty="true" />
-    <PackageReference Include="NServiceBus.Transport.SqlServer" Version="7.0.0-alpha.297" />
+    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="8.0.0-alpha.940" GeneratePathProperty="true" />
+    <PackageReference Include="NServiceBus.Transport.SqlServer" Version="7.0.0-alpha.336" />
     <PackageReference Include="NUnit" Version="3.13.1" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
   </ItemGroup>
 
   <ItemGroup>
     <Compile Remove="$(PkgNServiceBus_AcceptanceTests_Sources)\**\*.cs" />
-    <Compile Include="$(PkgNServiceBus_AcceptanceTests_Sources)\contentFiles\cs\any\**\EndpointTemplates\*.cs" />
-    <Compile Include="$(PkgNServiceBus_AcceptanceTests_Sources)\contentFiles\cs\any\**\ScenarioDescriptors\*.cs" />
-    <Compile Include="$(PkgNServiceBus_AcceptanceTests_Sources)\contentFiles\cs\any\**\NServiceBusAcceptanceTest.cs" />
+    <Compile Include="$(PkgNServiceBus_AcceptanceTests_Sources)\contentFiles\cs\$(TargetFramework)\**\EndpointTemplates\*.cs" />
+    <Compile Include="$(PkgNServiceBus_AcceptanceTests_Sources)\contentFiles\cs\$(TargetFramework)\**\ScenarioDescriptors\*.cs" />
+    <Compile Include="$(PkgNServiceBus_AcceptanceTests_Sources)\contentFiles\cs\$(TargetFramework)\**\NServiceBusAcceptanceTest.cs" />
```

Comment:
Anyone know why this changed, and if it's valid?

Reply:
It's because:

1. The AT Sources package is monolithic and should be split into multiple packages. This junk is to get just those 3 support files but none of the regular acceptance tests themselves.
2. Particular.Packaging had to change how it bundles the assets in a NuGet package in order to support working correctly with the new mix of target frameworks we're supporting, not working if you try to do netstandard2.0 with net461 or something like that, so the package itself no longer contains an "any" directory, so this is just fixing the paths to find the right files in the right places.